debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -94,37 +94,77 @@ public static String getClassName(J9IndexableObjectPointer objPointer) throws Co
 
 	public static U32 rawSize(J9IndexableObjectPointer objPointer) throws CorruptDataException
 	{
+		boolean isIndexableDataAddrPresent = false;
```

Comment:
Please remove redundant initializers like this (the code will compile only if the compiler can prove it is definitely assigned before each use). Also line 135.